src/components/FullRecipe/FullRecipe.tsx

```diff
@@ -1,42 +1,41 @@
 import React, { useState } from "react";
-import ShowButton from "../Button/ShowButton";
-import Ingridient from "../Ingridient/Ingridient";
+import Ingredient from "./Ingredient/Ingredient";
 import classes from "./FullRecipe.module.css";
 import { flipInY } from "react-animations";
 import styled, { keyframes } from "styled-components";
+import FullRecipeType from "../../types/FullRecipeType";
+import Button from "react-bootstrap/esm/Button";
 
-type Props = {
-  dish: any;
-  limit: number;
-};
 const flipInYAnimation = keyframes`${flipInY}`;
 const FlipInYDiv = styled.div`
   animation: 1.25s ${flipInYAnimation};
 `;
 
 const CHARACTER_LIMIT = 100;
 
-const FullRecipe = ({ dish, limit }: Props) => {
+const FullRecipe = ({ dish, limit }: FullRecipeType) => {
```

Comment:
or just eg: IProps :)
in general it's good to keep components props inside. You - probably - will never use this type anywhere else. if in multiple places - of course you should keep it in a separate file.